src/services/homarr-icon.service.ts

```
@@ -0,0 +1,111 @@
+import type { Icon } from "../models/Icon";
+import * as _iconDto from "./dto/IconDto";
```

ESLint

[error] '_iconDto' is defined but never used.

(@typescript-eslint/no-unused-vars)